starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -65,6 +65,40 @@ def _get_agasc_stars(ra, dec, roll, radius, date, agasc_file):
 
     return stars_dict
 
+import numpy as np
+import mica.stats.acq_stats
+import mica.stats.guide_stats
+
+ACQS = mica.stats.acq_stats.get_stats()
+GUIDES = mica.stats.guide_stats.get_stats()
+
+
+def get_mica_stats(agasc_id, time):
+
+    time = float(time)
+
+    acqs = ACQS[(ACQS['agasc_id'] == int(agasc_id))
+               & (ACQS['guide_tstart'] < time)]
+    ok = (acqs['img_func'] == 'star') & (~acqs['ion_rad']) & (~acqs['sat_pix'])
```

Comment:
It looks like in Py3 all these string columns are now bytestrings so this match on 'star' doesn't work.   Should try to fix this on the mica side.
```
In [23]: acqs['img_func']
Out[23]: 
array([b'star', b'NONE', b'star', b'star', b'star', b'NONE', b'star',
       b'star', b'star', b'star', b'star', b'star', b'star', b'star',
       b'star', b'star', b'star', b'star', b'star', b'star', b'star',
       b'star'], 
      dtype='|S7')
```

Reply:
If you turn it into an astropy `Table` then the compare will work.

Reply:
Wondering if I should do that on the mica side.  Also wondering if we have other codes that are going to be unhappy with this out of pytables.

Reply:
This is a key sticking point in Py3 migration, and one of the main drivers for https://github.com/astropy/astropy/pull/5700.  Takeaway is to use astropy `Table` when dealing with character data coming from HDF5.

Reply:
Upstream fix to use `Table` in mica seems like the right answer.  Probably that will break less code than not doing it.  Maybe??  :smile: